src/components/AttachmentCarousel/index.js

```diff
@@ -323,7 +335,6 @@ class AttachmentCarousel extends React.Component {
                         // promotes the natural feeling of swiping left/right to go to the next/previous image
                         // We don't want to invert the list for desktop/web because this interferes with mouse
                         // wheel or trackpad scrolling (in cases like document preview where you can scroll vertically)
-                        inverted={this.canUseTouchScreen}
```

Comment:
You can remove the above comments as they were related to this prop you removed.

Reply:
Maybe it's a good idea to move these comments to line 179, where we are now inverting manually